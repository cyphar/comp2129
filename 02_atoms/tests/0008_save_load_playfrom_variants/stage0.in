START 5 8 8
PLACE 0 1
PLACE 2 0
PLACE 3 1
PLACE 2 2
PLACE 7 7
DISPLAY
SAVE save0.as
PLACE 2 2
DISPLAY
SAVE save1.as
