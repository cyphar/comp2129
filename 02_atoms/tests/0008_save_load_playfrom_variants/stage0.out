Game Ready
Red's Turn

Green's Turn

Purple's Turn

Blue's Turn

Yellow's Turn

Red's Turn


+-----------------------+
|  |  |G1|  |  |  |  |  |
|R1|  |  |P1|  |  |  |  |
|  |  |B1|  |  |  |  |  |
|  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |Y1|
+-----------------------+

Game Saved

Cannot Place Atom Here


+-----------------------+
|  |  |G1|  |  |  |  |  |
|R1|  |  |P1|  |  |  |  |
|  |  |B1|  |  |  |  |  |
|  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |Y1|
+-----------------------+

Game Saved

